### DISCUSSION TITLE
Welcome to OpenWrt_R7800_Stable Discussions!

### DISCUSSION THREAD

**Discussion by yaya131** [yaya131/OpenWrt_R7800_Stable#10] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by jornersn** [comment]
大佬，请问，R7800的固件是不再更新了吗？

**Reply by tornadomord** [reply]
麻烦问问，E8450不能用了吗?
